Library/Homebrew/cmd/update-bash.sh

```diff
@@ -289,10 +301,16 @@ EOS
   # this procedure will be removed in the future if it seems unnecessary
   rename_taps_dir_if_necessary
 
+  DIRS=("$HOMEBREW_LIBRARY"/Taps/*/*)
+  if [[ -z "$SKIP_UPDATE_HOMEBREW_REPOSITOR" ]]
+  then
+    DIRS=("$HOMEBREW_REPOSITORY" "${DIRS[@]}")
+  fi
```

Comment:
I mean `DIRS="$DIRS $HOMEBREW_REPOSITORY"`. I'd really rather we avoid using Bash arrays; it'd be nice to avoid Bashisms (particularly those with different and somewhat confusing syntax) when there's no real benefit in them. It just makes understanding this script harder for people who have done shell scripting but not Bash.